finish feature unit test

namdhgc opened the pull request:
@jackdiff Please take a review for me when you have free time

namdhgc committed:
finish feature unit test

jackdiff commented on app/Http/Models/User.php:
```diff
@@ -17,6 +17,10 @@ class User extends Model
 
+	protected $fillable = [
+        'name', 'address', 'age', 'avatar_path'
+    ];
```
Please fix this indent

namdhgc replied:
In my sublime text, indent looks good @@

jackdiff commented on tests/Feature/UserControllerTest.php:
```diff
@@ -0,0 +1,569 @@
+
+	public function testAddUserFailWithConditionNameTooLong()
+	{
+		$Response 		= new Response();
```
Please fix the indent

namdhgc replied:
In my sublime text, indent looks good @@

jackdiff commented on tests/Feature/UserControllerTest.php:
```diff
@@ -0,0 +1,569 @@
+        ];
+        $Response 			= new Response();
+    	$base_response		= $Response->response( '200', Lang::get('message.web.success.0003'), $delete_request_array, true );
+        $response 			= $this->call('POST', 'user/add', $request_array);
```
Should assert adding new user success here before do delete

namdhgc replied:
added assert adding newuser success before delete

jackdiff commented on tests/Feature/UserControllerTest.php:
```diff
@@ -0,0 +1,569 @@
+        ];
+        $Response 			= new Response();
+    	$base_response		= $Response->response( '0001', Lang::get('message.web.error.0001'), "", false );
+        $response 			= $this->call('POST', 'user/add', $request_array);
```
Should assert adding new user success here before do delete

namdhgc replied:
added assert adding new user success before delete

jackdiff commented on tests/Feature/UserControllerTest.php:
```diff
@@ -0,0 +1,569 @@
+        $Response 			= new Response();
+    	$base_response		= $Response->response( '0001', Lang::get('message.web.error.0001'), "", false );
+        $response 			= $this->call('POST', 'user/add', $request_array);
+        $delete_response 	= $this->call('POST', 'user/delete', $delete_request_array);
```
Should assert is there any record that has id = 2 in database before deleting

namdhgc replied:
added field id in assert is there any record has id = 2 before deleting

namdhgc committed:
applied change as comment:
+ fix indent
+ add more assert

namdhgc committed:
applied change follow Mr. Truong comment:
+ toastr message show in the bottom-left
+ fix max height of textarea
+ validate message more insight
+ unit test add case assert insert success
